index.html

```diff
@@ -14,19 +14,19 @@
         <header class="header">
             <nav class="toolbar">
                 <a href="#">Edwin Gichuhi</a>
-                <img src="./images/Icon-Menu.png" alt="menu">
+                <img src="./images/Icon-Menu.png" alt="menu icon" id="hamburger-menu">
                 <ul class="menu-items">
                     <li>
-                        <a href="#">Portfolio</a>
+                        <a href="#portfolio">Portfolio</a>
                     </li>
                     <li>
-                        <a href="#">About</a>
+                        <a href="#about">About</a>
                     </li>
                     <li>
-                        <a href="#">Contact</a>
+                        <a href="#contact">Contact</a>
                     </li>
                     <li>
-                        <a href="#">

```

Comment:
- [Optional] Dear @egichuhis, Your navbar is looking great, One thing to improve is your font type should be the same as the fonts in the Figma design. 
- [Optional] Also I see there is not a `post` Icon as it is on the Figma design. please fix this for a better design,

## Your navbar
![Capture2](https://user-images.githubusercontent.com/69077061/204100274-ba330c9f-2072-4c1e-b693-e66447486a87.PNG)

## Figma navbar
![figma](https://user-images.githubusercontent.com/69077061/204100275-7eb5cfc4-baed-43ae-9224-fefcf5b7fb00.PNG)